data_structures/stacks/infix_to_prefix_conversion.py

```diff
@@ -134,24 +166,20 @@ def infix_2_prefix(infix):
         ...
     IndexError: list index out of range
 
-    >>> infix_2_prefix('())')  # doctest: +NORMALIZE_WHITESPACE
-     Symbol  |  Stack  | Postfix
-    ----------------------------
-       (     | (       |
-       (     | ((      |
-       )     | (       |
-             |         | (
-    '('
+    >>> infix_2_prefix('())')
+    Traceback (most recent call last):
+        ...
+    ValueError: invalid expression
     """
-    infix = list(infix[::-1])  # reverse the infix equation
+    new_infix = list(infix[::-1])  # reverse the infix equation
 
-    for i in range(len(infix)):
-        if infix[i] == "(":
-            infix[i] = ")"  # change "(" to ")"
-        elif infix[i] == ")":
-            infix[i] = "("  # change ")" to "("
+    for i in range(len(new_infix)):
+        if new_infix[i] == "(":
+            new_infix[i] = ")"  # change "(" to ")"
+        elif new_infix[i] == ")":
+            new_infix[i] = "("  # change ")" to "("
 
-    return (infix_2_postfix("".join(infix)))[
+    return (infix_2_postfix("".join(new_infix)))[
         ::-1
     ]  # call infix_2_postfix on Infix, return reverse of Postfix
```

Comment:
Let's not wrap lines of code just to fit in a comment.
```suggestion
    # call infix_2_postfix on Infix, return reverse of Postfix
    return (infix_2_postfix("".join(new_infix)))[::-1]
```